src/sparql/reports/obsoletiontagcheck.sparql

```diff
@@ -0,0 +1,382 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
```

Comment:
@sabrinatoro did you intend to commit this SPARQL query?

Reply:
Yes, I committed this sparql query because it is helpful to have when having a lot of obsoletion candidates and we want to check them before merging the ROBOT template (otherwise the ROBOT merging fail, one term at a time).